drivers/wifi/nrf700x/src/fw_load.c

```diff
@@ -66,28 +70,195 @@ LOG_MODULE_DECLARE(wifi_nrf, CONFIG_WIFI_NRF700X_LOG_LEVEL);
 INCBIN(_bin, nrf70_fw, STR(CONFIG_NRF_WIFI_FW_BIN));
 #endif /* CONFIG_NRF_WIFI_PATCHES_BUILTIN */
 
+#ifdef CONFIG_NRF_WIFI_PATCHES_EXT_FLASH_STORE
+#include <zephyr/fs/nvs.h>
```

Comment:
Why is NVS pulled as a dependency?

Reply:
Oops, left over code when I had used NVS first.